### ISSUE TITLE
Review unstructured.io for curation of data for LLM

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#12]
## User story
1. As a dev team
2. I need to review Marian's suggestion of unstructured.io for chunking and prepping of data
3. So that the uploaded documents can be prepped for use with the LLM framework

## Acceptance criteria
* Check licensing requirements of unstructured.io satisfy class requirements
* If licensing is acceptable, explore unstructured.io as a source to prep the data and how to use it

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
